### PR TITLE
Add Improvements to Pagerduty Plugin 

### PR DESCRIPTION
**Pull request by douglasread**
## Hey, I just made a Pull Request!

This change adds the ability for the Pagerduty Plugin to describe Change Events and link to Escalation policy. For more details: https://github.com/backstage/backstage/issues/6048. 

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/pagerduty/src/api/client.ts [hunk 1]
```diff
@@ -80,11 +92,11 @@ export class PagerDutyClient implements PagerDutyApi {
   }
 
   triggerAlarm({
-    integrationKey,
-    source,
-    description,
-    userName,
-  }: TriggerAlarmRequest): Promise<Response> {
+                 integrationKey,
```
Comment (jhaals) [inline plugins/pagerduty/src/api/client.ts:95]: Prettier is currently complaining about these lines.

Comment (douglasread) [reply plugins/pagerduty/src/api/client.ts]: fixed

**Comment by benjdlambert** [comment]
@douglasread are you interested in picking up this to finish it off?

**Comment by douglasread** [comment]
Hey @benjdlambert. I pushed changes to address the comments above and hopefully finish it off.

**Review by freben — APPROVED** [review]

**Comment by freben** [comment]
Approved! I took the liberty of pushing the final little fixup commit to hopefully make it green

**Pull request merged**